Training/Services/ImportService.cs

```diff
@@ -25,39 +25,38 @@ public ImportService(IClient client, string projectKey)
             _csvHelper = new CSVHelper();
         }
 
-        public async Task<ImportSink> CreateImportSink(ImportSinkDraft draft)
+        public async Task<ImportContainer> CreateImportContainer(ImportContainerDraft draft)
         {
-            var importSink = await _importClient.WithImportApi().WithProjectKey(_projectKey)
-                .ImportSinks()
+            var importContainer = await _importClient.WithImportApi().WithProjectKeyValue(Settings.ProjectKey)
+                .ImportContainers()
                 .Post(draft)
-                .ExecuteAsync();
-            return importSink;
+                .ExecuteAsync() as ImportContainer;
+            return importContainer;
         }
 
-        public async Task<ImportOperation> CheckImportOperationStatus(string importSinkKey, string id)
+        public async Task<ImportOperation> CheckImportOperationStatus(string importContainerKey, string id)
         {
-            var importOperation = await _importClient.WithImportApi().WithProjectKey(_projectKey)
-                .ProductDrafts()
-                .ImportSinkKeyWithImportSinkKeyValue(importSinkKey)
+            var importOperation = await _importClient.WithImportApi().WithProjectKeyValue(_projectKey)
                 .ImportOperations()
                 .WithIdValue(id)
-                .Get().ExecuteAsync();
+                .Get().ExecuteAsync() as ImportOperation;
```

Comment:
@nagesh-dixit I think we don't need casting here, we can change it to return Task<IImportOperation>